recipes/modern-cpp-kafka/all/test_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def requirements(self):
+        self.requires("librdkafka/1.8.2")
```

Comment:
If `modern-cpp-kafka` requires `librdkafka`, it should be a requirement of the recipe itself (yes, even if it is a header only package), that way the user consuming `modern-cpp-kafka` has everything needed (see `approvaltests.cpp`, it is the same pattern).

If the consumer wants to override the `librdkafka` they can always do it explicitly:

```python
def requirements(self):
    self.requires("librdkafka/<user-overrides>")
    self.requires("modern-cpp-kakfa/2022.06.15")
```

but if they are fine with the hardcoded version, they can just require what they directly need:

```python
def requirements(self):
    self.requires("modern-cpp-kakfa/2022.06.15")
```

Reply:
@jgsogo
OH, users are free to override the dependencies explicitly? That's flexible for conan! Let me update the PR. Thanks!